From c674556438f0829230f71e3b022cc48102efe60c Mon Sep 17 00:00:00 2001
From: Julien Gilli <julien.gilli@joyent.com>
Date: Tue, 24 Oct 2017 11:06:03 -0700
Subject: [PATCH] VOLAPI-88 do not refer to joyent/node-triton's master in
 installation instructions Reviewed by: Josh Wilsdon <jwilsdon@joyent.com>
 Approved by: Josh Wilsdon <jwilsdon@joyent.com>

---
 README.md | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/README.md b/README.md
index 213ac85..547275c 100644
--- a/README.md
+++ b/README.md
@@ -5,7 +5,7 @@
 -->
 
 <!--
-    Copyright (c) 2016, Joyent, Inc.
+    Copyright (c) 2017, Joyent, Inc.
 -->
 
 # sdc-volapi
@@ -67,11 +67,10 @@ use volumes, just not through Docker APIs/clients.
   should be installed. By default, the installation process will use `coal` as
   the datacenter name.
 
-3. Install an experimental version of node-triton with NFS shared volumes
-   support:
+3. Install the latest version of node-triton to get NFS shared volumes support:
 
    ```
-   $ npm install ssh+git@github.com:joyent/node-triton.git#tritonnfs
+   $ npm install -g triton
    ```
 
 #### Disclaimer
-- 
2.21.0

